### ISSUE TITLE
Text in all caps - SiteImprove - Stat and Table

### ISSUE THREAD

**Issue by joewhitsitt** [uiowa/uiowa#9097]
Siteimprove is flagging an issue with the stat component and html tables

# Text in all caps
The text-transform property has been used to display an entire paragraph in uppercase (capital letters).

# Proposed solution - https://github.com/uiowa/uiowa/issues/9097#issuecomment-3403015443

- Update stat component and table component in uids
- stat uses `<span>` and not `<p>`
- tables should use sentence case
- pull updated components into SiteNow
- Update hook to move long text to text
- verify SiteImprove is no longer reporting issues with stat and table

**Comment by briand44** [comment]
table headers being flagged as well, was mentioned on https://github.com/uiowa/uiowa/issues/8508#issuecomment-3089951112. Not sure if we have a separate issue for that but mentioning it here so it doesn't get lost.

**Comment by briand44** [comment]
## Grooming
 
**Quadrant:** 

Complicated - since we want to address in both uids and uids_base

 **Discussion:**
 - We will need to address uids first
 - address in uids_base as well in this slice
 - Capitalize initial letter and lowercase the rest
 - Address both stat and table as part of this
 - should be addressing in the text-transform property

**Proposed solution or definition of done:** 

- UIDS stats and tables have first letter capitalized and rest lower
- SiteNow is pulling these updated components
- SiteImprove is no longer flagging these as issues - should be able to test with SI browser plugin

**Comment by bspeare** [comment]
The brand team has a few concerns about the Antonio adjustments and we might want to postpone this until we can discuss it further.

**Comment by joewhitsitt** [comment]
ack! okay, this is taking a bit longer than i thought. using Ben's sitenow_update_9047 and basically trying to singlefy it and inverse it to switch to `string`. my update hook doesn't currently error out (good), but doesn't switch to a text field (bad), doesn't insert the data (bad), and doesn't touch the revisions table which seems to be hashed like `block_content_r__575dc17943` (ugh)

**Comment by bspeare** [comment]
Is there a Siteimprove rule beyond https://alfa.siteimprove.com/rules/sia-r72 for caps or is that the extent of their guidance?
